src/main/java/reposense/RepoSense.java

```diff
@@ -76,6 +84,22 @@ public static void main(String[] args) {
         }
     }
 
+    /**
+     * Returns a list of relevant repo folders to be zipped up later
+     * @param sourcePath
+     * @param configs
+     * @return
+     */
+    private static HashSet<Path> getRelativeFolders(Path sourcePath, List<RepoConfiguration> configs) {
```

Comment:
Can be place in `FileUtil`. Since its being used by a method in that class.

Also, these remove these trivial JavaDocs.
We don't use them like this. Use it like how you did in another method.
```
     * @param sourcePath
     * @param configs
     * @return
```